src/utils.ts

```diff
@@ -111,7 +112,11 @@ export abstract class AuthenticatedService<Api extends $protobuf.rpc.Service> {
             new grpc.Client(host, grpc.credentials.createInsecure());
         const rpcImpl: $protobuf.RPCImpl = (method, requestData, callback) => {
             const path = `/${this.name}/${method.name}`;
-            client.makeUnaryRequest(path, _.identity, _.identity, requestData, this.metadata, null, callback);
+            if (this.metadata) {
+                client.makeUnaryRequest(path, _.identity, _.identity, requestData, this.metadata, callback);
+            } else {
+                client.makeUnaryRequest(path, _.identity, _.identity, requestData, callback);
+            }
```

Comment:
We can keep here single line:
```ts
client.makeUnaryRequest(path, _.identity, _.identity, requestData, this.metadata, {}, callback);
```

Reply:
No, we can't really:

![image](https://user-images.githubusercontent.com/128121/119511626-cefa7380-bd9c-11eb-8ae6-00a71354f7cb.png)

Reply:
Ok 👍